app/src/main/java/com/hedvig/app/feature/offer/ui/composable/insurely/RetrievedInfo.kt

```diff
@@ -0,0 +1,132 @@
+package com.hedvig.app.feature.offer.ui.composable.insurely
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.ContentAlpha
+import androidx.compose.material.Divider
+import androidx.compose.material.LocalContentAlpha
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.hedvig.app.R
+import com.hedvig.app.feature.offer.ui.OfferModel
+import com.hedvig.app.ui.compose.theme.HedvigTheme
+import com.hedvig.app.util.compose.preview.previewData
+import javax.money.MonetaryAmount
+
+@Composable
+fun RetrievedInfo(data: OfferModel.InsurelyCard.Retrieved) {
+    Column(
+        horizontalAlignment = Alignment.CenterHorizontally,
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(16.dp)
+    ) {
+        if (data.savedWithHedvig != null) {
+            SavedWithHedvigChip(data.savedWithHedvig)
+            Spacer(Modifier.height(6.dp))
+        }
+        Spacer(Modifier.height(8.dp))
+        CompositionLocalProvider(LocalContentAlpha provides ContentAlpha.medium) {
+            Text(
+                text = if (data.currentInsurances.size > 1) {
+                    LocalContext.current.resources.getQuantityString(
+                        R.plurals.offer_switcher_title,
+                        data.currentInsurances.size
+                    )
+                } else {
+                    "YOUR INSURANCE${data.insuranceProvider?.let { " WITH $it" }}"
+                },
+                style = MaterialTheme.typography.caption
+            )
+        }
+        Spacer(Modifier.height(24.dp))
+        Text(
+            text = "${data.totalNetPremium}", // todo locale formatting
```

Comment:
Will depend how the design ends up with. And if we want to force the currency before/after the number or if we can let it be as the current locale likes it